core/modules/admin_bar/css/admin_bar.css

```diff
@@ -78,6 +78,11 @@
   filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#9F9F9F', endColorstr='#808080',GradientType=0); /* IE6-9 */
 }
 
+#admin-bar li {
```

Comment:
This should be fine I think, if Admin Bar needs 0 margin/padding and previously it was just assuming it had none (perhaps provided by a theme or system.theme.css) then this could be a consistency improvement.